aptos-move/vm-genesis/src/lib.rs

```diff
@@ -317,23 +334,69 @@ fn create_and_initialize_validators(
     );
 }
 
-/// Publish the standard library.
-fn publish_stdlib(session: &mut SessionExt<impl MoveResolver>, stdlib: Modules) {
-    let dep_graph = stdlib.compute_dependency_graph();
+/// Publish all modules that should be available after genesis.
+fn publish_stdlib(session: &mut SessionExt<impl MoveResolver>, stdlib: Vec<CompiledModule>) {
+    let map = construct_module_map(stdlib);
+    let root_address = AccountAddress::from_hex_literal("0x1").unwrap();
```

Comment:
nit: Can we call this aptos_framework_address? Let's not mention root anywhere that's not part of test/test code if possible

Reply:
updated